_pages/en/it-picture-medium-term.md

```diff
@@ -132,11 +146,20 @@ En conclusion, ...
 - https://cloud.google.com/solutions/devops/devops-measurement-wip-limits
 - https://cloud.google.com/solutions/devops/devops-measurement-visual-management
 
+Business lines understand their accountability in sustaining the IT systems used to enable their business.
+This means part of their budget is set to assure technical debt is measured, exposed to the business line, and handled in a timely manner
+
+Operations have moved towards a Site Reliability Engineering (SRE) practice where operations is treated like a software practice.
+ESDC applications are managed, maintained, and monitored using software.
+SREs collect data throughout the IT value chain and aggregate it to different audiences.
+
 Teams have established Work in progress (WIP) limits based on team capacity and workload assigned, in order to prioritize and focus on completing a small number of high-priority tasks. Once the process is implemented accurately, this will result in faster lead times, higher quality, lower costs, and less waste.
 
-The IT branch is collecting data from key areas and appropriate metrics throughout the value chain and ensuring that the right data is being collected.The collected data is available, shared, and used to make decisions across ESDC (not just IT).
-Business lines performance metrics are used as benchmark and the accountability of IT's impact to ESDC's mandate is shared.
-Teams setup proactive notification that generate alert notifications when monitored values approach known failure thresholds.
+SRE provides proactive failure notification, using key indicators and thresholds in their monitoring software, and get alerted before an application fails to pro-actively resolve issues before they impact users.
```

Comment:
We should just say IT gets proactive failure notification, ...

Reply:
reworded